_tools/meta.ts

```diff
@@ -0,0 +1,45 @@
+import { BuildOptions } from "https://deno.land/x/dnt@0.30.0/mod.ts";
+
+export const makeOptions = (version: string): BuildOptions => ({
+  test: false,
+  shims: {
+    undici: true,
+    custom: [{
+      module: "urlpattern-polyfill",
+      globalNames: ["URLPattern"],
```

Comment:
Codacy has a fix for the issue: [Replace `······` with `········`](https://app.codacy.com/gh/httpland/http-router/pullRequest?prid=9963312)
```suggestion
        globalNames: ["URLPattern"],
```